Sources/App/CRPFields.swift

```diff
@@ -0,0 +1,78 @@
+import StevensonCore
+import Vapor
+
+public extension JiraService {
+
+    /// A CRPIssue is a Jira issue specific to our CRP Board (aka Releases Plan Board)
+    /// CRP means "Change Request Process" and is part of our SSDLC to track upcoming releases
+    public typealias CRPIssue = JiraService.Issue<JiraService.CRPIssueFields>
+
+    public struct CRPIssueFields: JiraIssueFields {
+
+        // MARK: Custom Field Types specific to CRP board
+
+        public struct Environment: Content {
+            let id: String
+
+            static let playStore = Environment(id: "12394")
+            static let appStore = Environment(id: "12395")
+            static let notApplicable = Environment(id: "12396")
+        }
+
+        struct InfoSecStatus: Content {
+            let id: String
+            static let yes = InfoSecStatus(id: "11942")
+            static let no = InfoSecStatus(id: "11941")
+        }
+
+        // MARK: Fields
+
+        public let project = FieldType.ObjectID(id: "13402") // CRP Project
+        public let issueType = FieldType.ObjectID(id: "11439") // "CRP: Code Change Request"
+        let summary: String
+        var changelog: FieldType.TextArea.Document
+        let environments: [Environment]
+        var businessImpact: FieldType.TextArea.Document
+//        let jiraReleaseURL: String
+//        let githubReleaseURL: String
+        var testing: FieldType.TextArea.Document
+        var accountablePerson: FieldType.User
+        let infoSecChecked: InfoSecStatus
+
+        // MARK: Content keys
+
+        enum CodingKeys: String, CodingKey {
+            case project = "project"
+            case issueType = "issuetype"
+            case summary = "summary"
+            case changelog = "customfield_12537"
+            case environments = "customfield_12592"
+            case businessImpact = "customfield_12538"
+//            case jiraReleaseURL = "customfield_12540"
+//            case githubReleaseURL = "customfield_12541"
```

Comment:
Note: There are currently commented out because I can't find a way to send those two fields via the JIRA API.

These two fields are of type "URL" in our JIRA project, but every way I tried to send them (and I tried a lot 😓), the API rejected those fields (and when I asked others who manipulated the same API they couldn't find what was wrong in my request either 😢, and the JIRA API documentation is not helping either)

I'm hoping we find a way to send those two via the API one day, but I've lost too much time trying to debug those two, so figured we'll keep that for later

Reply:
Can you create a ticket and leave a todo not here `// [TICKET-XXX] TODO: fix it`